app/src/features/surveys/animals/animal-form/create/CreateAnimalPage.tsx

```diff
@@ -25,6 +25,7 @@ import { Link as RouterLink } from 'react-router-dom';
 export const defaultAnimalDataFormValues: ICreateEditAnimalRequest = {
   nickname: '',
   species: null,
+  sex: AnimalSex.UNKNOWN,
```

Comment:
I assume eventually we'd want to fetch the list of allowed sex values from Critterbase?